### DISCUSSION TITLE
Is the tailwindcss.com website not available in the public repo?, i need to make it reference. Thanks

### DISCUSSION THREAD

**Discussion by iamhiwon** [tailwindlabs/tailwindcss#5235] [Help]
Is the tailwindcss.com website not available in the public repo?, i need to make it reference.
Thanks

**Comment by simonswiss** [comment]
Hey!

Yes, it's available, but it's a different repo than Tailwind CSS itself:

https://github.com/tailwindlabs/tailwindcss.com
